examples/kfto-sft-feast-rag/feast_rag_retriever.py

```
@@ -0,0 +1,334 @@
+from abc import ABC, abstractmethod
+from typing import Callable, Dict, List, Optional, Union, Tuple, Any
+import numpy as np
+import torch
+from feast import FeatureView
+from feast.online_response import OnlineResponse
+from pyarrow import float16
```

Ruff (0.11.9)

`pyarrow.float16` imported but unused

Remove unused import: `pyarrow.float16`

(F401)